core/components/seosuite/src/Plugins/Resource.php

```diff
@@ -61,7 +61,7 @@ protected function isEnabled($resource, $mode = 'create')
         }
 
         if (count($templates) >= 1) {
-            return in_array($template, $templates, true);
+            return !in_array($template, $templates, true);
```

Comment:
Logic was opposite, it was only enabled in case the current template was in the "disabled templates". Maybe renaming `$templates` to `$disabledTemplates` would probably better reflect the intention/code readability.